### ISSUE TITLE
Instrument menu

### ISSUE THREAD

**Issue by goonstation-issuebot** [goonstation/goonstation#14239]
### Labels

[BUG][MINOR]

### Description

The menu that we see when we use instruments like triangle or guitar will not go away

### Steps to reproduce

tocuh a character taht is surgicaly implanted with instrument
menu gets stuck and never goes away

### Expected Behavior

it should not open up a menu, since we cant use instrumens inplanted unless horn or something

### Additional Information & Screenshots

Reported by: Wantilathor
On server: Goonstation 3 Roleplay: Morty
Active test merges: nothing
Round log date: 2023-05-25-22-00
Reported on: 2023-05-25 23:01:58
Map: COGMAP2

**Comment by TealSeer** [comment]
Duplicate of #14240
